components/collector/src/source_collectors/dependency_track/source_up_to_dateness.py

```diff
@@ -87,3 +91,21 @@ def _timestamp_to_datetime(timestamp: int | None) -> datetime | None:
     def _is_latest(project: DependencyTrackProject) -> bool:
         """Return whether the project is the latest version."""
         return project.get("isLatest", False)
+
+    def _up_to_dateness(self, project: DependencyTrackProject) -> str:
+        """Return the up-to-dateness of the project."""
+        date_times = self._project_event_datetimes(project)
+        if not date_times:
+            return "unknown"
+        min_date_time = self.days(self.minimum(date_times))
```

Comment:
Why do we compare on the minimum? I'd expect the latest event on a project to be polled, or are these project events somehow overwritten with each SBOM import and/or analysis?

Reply:
Because DT has two event types for each project: the last BOM-import and the last BOM-analysis. For each event type there is one date. So `date_times` is at most two dates. If the user wants to take both event types into account for the up-to-dateness, it makes sense to look at the minimum, because that is the age of the last analysis of the last imported BOM.

Reply:
Right, makes sense when known. Perhaps worth adding an inline comment or docstring to explain